.editorconfig

```diff
@@ -1,28 +0,0 @@
-# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
Do we need to drop this file?

Reply:
> > I went through the steps to set up and run template, but node complains that it cannot resolve modules in `packages/*.
> 
> I resolved this by running `npm link @geostreams/core` in `packages/geostreaming`. I didn't notice it in readme and might want to add something there.

Thanks! Shouldnt lerna link the subpackages automatically? Or is there any config for it that you had done for the previous setup?